applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -866,7 +866,8 @@ ZBUS_LISTENER_DEFINE(sdu_ref_msg_listen, audio_datapath_sdu_ref_update);
 int audio_datapath_pres_delay_us_set(uint32_t delay_us)
 {
 	if (!IN_RANGE(delay_us, CONFIG_AUDIO_MIN_PRES_DLY_US, CONFIG_AUDIO_MAX_PRES_DLY_US)) {
-		LOG_WRN("Presentation delay not supported: %d", delay_us);
+		LOG_WRN("Presentation delay not supported: %d us", delay_us);
+		LOG_WRN("Keeping current value: %d us", ctrl_blk.pres_comp.pres_delay_us);
```

Comment:
There are only log changes in this PR. No handling required?

Reply:
Correct. Since the default is already used, and then you are not allowed to set PD=0 it will continue to use the default value. So I just removed the error print (as that would only be printed if someone attempts to set PD=0) and then printed what the default value is